x11-terms/terminology/terminology-1.1.0.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
oooold (see 7)

Reply:
Oh right, I thought I need xdg-utils for xdg_environment_reset, but now I remember EAPI7 takes care of that. Xdg-utils isn't EAPI7 compatible. 

Updating in a minute, cheers!